internal/resources/policy/kind/network/constants.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package policykindnetwork
+
+import (
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
+	reciperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/network/recipe"
+)
+
+const (
+	ResourceName = "tanzu-mission-control_network_policy"
+	typePolicy   = "network-policy" // Type of Policy as defined in API
```

Comment:
nit: Lets rename it as policyKind just to keep the naming consistent

Reply:
This is named `typePolicy` to keep it consistent with other policy types. Ref: [security policy](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/adfe80d8b841aeb281132346c0df759036bc127a/internal/resources/policy/kind/security/constants.go#L15)